config.toml

```
@@ -143,16 +143,18 @@ theme = "elate"
       alt = "Sébastien Rémond"
 
     [[params.testimonials.item]]
-      quote = "&ldquo;Par la justesse de ses reformulations et feedbacks, Tristan a réussi à susciter au sein du groupe un changement de regard et l’envie d’oser'.&rdquo;"
-      person = 'O.L, <a href="http://franceurbaine.org" target="_blank"> France Urbaine </a> <span class="subtext">Directeur Général </span>'
+      quote = "&ldquo;Tristan a fait preuve d'une grande intelligence des situations dans un contexte de transformation majeure de l'entreprise
+      .&rdquo;"
+      person = 'Catherine Belotti, <a href="" target="_blank">
+      </a> <span class="subtext">Directrice innovation sociale Schneider Electric  </span>'
```

```suggestion
      <span class="subtext">Directrice innovation sociale Schneider Electric  </span>'
```

Suppression de la balise fermante du lien